server/api/user/index.ts

```diff
@@ -1,13 +1,16 @@
-import { Express } from 'express'
 import { ApiEndPoint } from '@common/api/endpoint'
-import { getInvitation } from './getInvitation'
+import { Express } from 'express'
+
 import { getAcceptInvitation } from './acceptInvitation'
+import { getInvitation } from './getInvitation'
+import { getMany } from './getMany'
 import { getProfilePicture } from './getProfilePicture'
 
 export const UserApi = {
   init: (express: Express): void => {
     express.get(ApiEndPoint.User.getByInvitation(), getInvitation)
     express.get(ApiEndPoint.User.acceptInvitation(), getAcceptInvitation)
     express.get(ApiEndPoint.User.getProfilePicture(), getProfilePicture)
+    express.get(ApiEndPoint.User.many(), getMany)
```

Comment:
We do need an auth middleware 

Only users that have a role in that county can see the users .

Do you want to add it in this pr or next pr ?

Reply:
nope, if it's only for the country it's easy to do